otelcollector/prometheusreceiver/config.go

```diff
@@ -18,7 +18,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"io"
+	"net/url"
```

Comment:
grace, can you add it somewhere the version & commit we have been synced to for prom receiver ?

Reply:
Right, thanks I updated the files we have for prom-version and otelcollector version. The prom receiver is synced to the v0.66.0 tag (same as the otelcollector version) so there are no conflicts